### PR TITLE
Adding Ingredient Class

### PR DESCRIPTION
**Pull request by rauljch9**
This is the class that wil be use in Inventory and Products

### CODE REVIEW

**Comment by reynoldmorel** [comment]
Could you rename branch `main` to `master`? Could you also improve your commit history? It says you adding ingredient and also adding inventory. Could separate in different PRs?

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# SoftFoodWare
+# Description
+
+# Architecture
+This project will be a software for restaurants, this software will handle General Orders, Delivery Orders, among others.
+
+The architecture is:
+
+<img src="\UML Diagram.jpg">
```
Comment (reynoldmorel) [inline README.md]: I can't see your image.

File: src/model/inventory/Ingredient.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package model.inventory;
+
+/**
+ * This class will be use in the inventory and wil be use in products for the preparation
```
Comment (reynoldmorel) [inline src/model/inventory/Ingredient.java]: typo, instead couldbe: `This class will be used by the inventory and will be used and final product preparation`.

File: src/model/inventory/Ingredient.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ */
+public class Ingredient{
+    int id;
```
Comment (reynoldmorel) [inline src/model/inventory/Ingredient.java:7]: Getters and Setters are missing.

File: src/model/inventory/InventoryEntry.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package model.inventory;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class wil be used to create documents that add stock to the Inventory
```
Comment (reynoldmorel) [inline src/model/inventory/InventoryEntry.java:7]: Could we add thsi in a separate PR

File: src/model/inventory/InventoryOutput.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package model.inventory;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class wil be used to create documents that remove stock to the Inventory
```
Comment (reynoldmorel) [inline src/model/inventory/InventoryOutput.java:7]: Could you ellaborate a bit more on what is this for?

Please correct the typos, use google translate if necessary.

File: src/model/inventory/Ingredient.java (hunk 2 above)
Comment (rauljch9) [reply src/model/inventory/Ingredient.java]: Javadoc Fixed!

File: src/model/inventory/Ingredient.java (hunk 3 above)
Comment (rauljch9) [reply src/model/inventory/Ingredient.java:7]: The getters and Setters are now added

**Comment by rauljch9** [comment]
I'm doing it all again in another branch

**Pull request closed without merging**